app/javascript/components/remove-catalog-item-modal.jsx

```diff
@@ -68,6 +69,7 @@ class RemoveCatalogItemModal extends React.Component {
          service_type: catalogItem.service_type, // 'atomic' or 'composite'
          services:     catalogItem.services})))
       .then(data => this.setState({data}))
+      .then(() => this.setState({loaded: true}))
```

Comment:
maybe this could be

`.then((data) => this.setState({ ...data, loaded: true }))`

Reply:
Done.